front/src/components/boxs/device-in-room/device-features/BinaryDeviceFeature.jsx

```diff
@@ -1,6 +1,21 @@
+import { Text } from 'preact-i18n';
+import cx from 'classnames';
+import { DEVICE_FEATURE_CATEGORIES, DEVICE_FEATURE_TYPES } from '../../../../../../server/utils/constants';
+
+const BINARY_CATEGORIES_TYPES_CUSTOM = {
+  [DEVICE_FEATURE_CATEGORIES.ELECTRICAL_VEHICLE_COMMAND]: [
+    DEVICE_FEATURE_TYPES.ELECTRICAL_VEHICLE_COMMAND.ALARM,
+    DEVICE_FEATURE_TYPES.ELECTRICAL_VEHICLE_COMMAND.LOCK
+  ]
+};
+
 const BinaryDeviceType = ({ children, ...props }) => {
+  const { category, type, last_value: lastValue } = props.deviceFeature;
+  const customText =
```

Comment:
One other option would be to simply check into the dictionary if the custom string exists, so there is no need to add another variable here! (check for `this.props.intl.dictionary` in the code)